lib/reverse_sentence.rb

```diff
@@ -1,6 +1,52 @@
 # A method to reverse the words in a sentence, in place.
-# Time complexity: ?
-# Space complexity: ?
+# Time complexity: O(n); Reverses the string - O(n), iterates through each character in string - O(n) to identify individual words
+# and then reverses the word - O(n).
+# Space complexity: O(1) - modifies the string in-place and creates the same number of variables regardless of string size.
```

Comment:
This is actually O(k) where k is the length of a word, but if the words are going to be relatively small you could call it O(1).